tests/test_models_circuit_backpropagation.py

```diff
@@ -8,9 +8,9 @@
 
 def construct_tensorflow_backend():
     try:
-        from qibo.backends import construct_backend
+        from qibo.backends import MetaBackend
 
-        backend = construct_backend("tensorflow")
+        backend = MetaBackend.load("tensorflow")
```

Comment:
Here, and in the other tests, I'd keep the `construct_backend` function.

The reason is partly the one above, but even that we can simply avoid to break, and keep using the old function. Since, in this case, it costs nothing to us, I'd just do that.